docs/src/main/sphinx/release/release-393.md

```diff
@@ -0,0 +1,120 @@
+# Release 393 (10 August 2022)
+
+## General
+
+## Security
+
+## Web UI
+
+## JDBC driver
+
+## Server RPM
+
+## Docker image
+
+## CLI
+
+## Accumulo connector
+
+## Atop connector
+
+## BigQuery connector
+
+## Blackhole connector
+
+## Cassandra connector
+
+## ClickHouse connector
+
+* Improve performance for queries with selective joins through push down of
+  dynamic filters to the data source. ({issue}`13334`)
+
+## Delta Lake connector
+
+## Druid connector
+
+* Improve performance for queries with selective joins through push down of
+  dynamic filters to the data source. ({issue}`13334`)
+
+## Elasticsearch connector
+
+## Google Sheets connector
+
+## Hive connector
+
+* Fix bug where query constraints were not used when fetching table statistics. ({issue}`13488`)
```

Comment:
What's the user-visible effect of this? Suboptimal performance? Correctness? Failures? It should be rephrased in those terms.

Reply:
cc: @alexjo2144 @findepi

Reply:
Increased query planning and execution time in certain cases.

Reply:
Tried again, let me know if it's good.

Reply:
Given the above, it should be:

> Improve performance when retrieving table statistics from the Metastore.

Reply:
It's not just performance. In the older code if you asked for table stats with a partition filter applied it'd not apply the partition filter when fetching stats. This does impact performance but also means that stats returned for table handles which had filters applied would be incorrect.

Reply:
That would just result in a slow performing query, though.